docs/stack/get-started/om-clients/stack-dotnet.md

```diff
@@ -41,7 +42,11 @@ Then open the `Redis.OM.Skeleton.csproj` file in your IDE of choice.
 
 ## Configure the app
 
-Add a `"REDIS_CONNECTION_STRING" field to your `appsettings.json` file to configure the application. Set that connection string to be the URI of your Redis instance. If using the docker command mentioned earlier, your connection string will be `redis://localhost:6379`.
+Add a `REDIS_CONNECTION_STRING` field to your `appsettings.json` file to configure the application. Set that connection string to be the URI of your Redis instance. If using the docker command mentioned earlier, your connection string will be `redis://localhost:6379`.
+
+### Connection string specification
```

Comment:
Maybe just point at the [Redis URI](https://www.iana.org/assignments/uri-schemes/prov/redis) IANA proposed spec? Or perhaps the one we maintain in the [specs repo](https://github.com/redis/redis-specifications/tree/master/uri)?

Reply:
Thanks, @slorello89 . The tricky bit is that the colon is required even if there's no username present. It's not clear to me that either of those two references captures that.

[For reference](https://github.com/redis/redis-om-dotnet/blob/968fa767fa7e4362785c4f58905071b23bc0d917/src/Redis.OM/RedisUriParser.cs#L74).

Reply:
Technically, those are standard [URI semantics](https://datatracker.ietf.org/doc/html/rfc3986#section-3.2.1) for the Authority section (in this case the user is an empty string). However I see your point, that it is somewhat confusing (probably why I put that error in there :) ). Perhaps link to the URI spec with a note that you need to use a `:password`?

Reply:
Also worth noting that `default:password` would also work if there is no username.

Reply:
Fair enough. I'll do as you ask. Look for another commit soon.